Src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmProcess.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#region Assembly Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
+// References\Debugger\v2.0\Microsoft.VisualStudio.Debugger.Engine.dll
+
+#endregion
+
+namespace Microsoft.VisualStudio.Debugger
+{
+    public class DkmProcess
+    {
+        public readonly DkmEngineSettings EngineSettings = new DkmEngineSettings();
+
+        private bool _nativeDebuggingEnabled;
+
+        public DkmProcess()
+        {
+        }
+
+        public DkmProcess(bool enableNativeDebugging)
+        {
+            _nativeDebuggingEnabled = enableNativeDebugging;
+        }
+
+        public DkmRuntimeInstance GetNativeRuntimeInstance()
```

Comment:
Unfortunately, no.  That would be a breaking API change with previous versions of VS.  You can tell this API was originally intended to be called from native code.